app/src/main/java/com/example/plannr/admin/adminEdit/FirebaseCallback2.java

```diff
@@ -0,0 +1,7 @@
+package com.example.plannr.admin.adminEdit;
+
+import java.util.HashMap;
+
+public interface FirebaseCallback2 {
```

Comment:
why is this FirebaseCallback2? what happened to the first one?

Reply:
FirebaseCallback2 contains a separate function that deals with getting the course selected for editing from the admin landing page. Once a course is selected from the landing page, the course code is pushed to the db under the selection table. Then, the edit fragment will read it from the database. It's essentially a way for the two fragments to communicate as from what I searched up, you can't directly pass information between them.